src/writers/post_operator/mod.rs

```diff
@@ -0,0 +1,11 @@
+// use glob::glob;
+use std::path::Path;
+use std::process::Command;
+
+pub fn format_all_rs(path: &Path) -> std::io::Result<()> {
```

Comment:
getting passed in a path when working with mostly strings as top level dirs is kinda greedy and puts a lot of weight on the caller, a fix could be: 

```rust
format_all_rs(path: &Path) -> 
    ...
```
 -> 
```rust
format_all_rs(path_string: &String) -> 
    path = Path::new(path_string);
    ...
```